paper/paper.bib

```diff
@@ -0,0 +1,66 @@
+@misc{diaz-vico+ramos-carreno_2022_scikitdatasets,
+  title = {{{scikit-datasets}}: {{Scikit-learn-compatible}} Datasets},
+  author = {{D{\'i}az-Vico}, David and {Ramos-Carre{\~n}o}, Carlos},
+  year = {2022},
+  month = mar,
+  doi = {10.5281/zenodo.6383047},
+  url = {https://github.com/daviddiazvico/scikit-datasets},
+  copyright = {MIT}
+}
+
+@misc{fajardo_2024_pyreadr,
+  title = {Pyreadr},
+  author = {Fajardo, Otto},
+  year = {2024},
+  month = jul,
+  doi = {10.5281/zenodo.13132498},
+  url = {https://github.com/ofajardo/pyreadr}
+}
+
+@misc{gautier_2024_rpy2,
+  title = {Rpy2: {{R}} in {{Python}}},
+  author = {Gautier, Laurent},
+  year = {2024},
+  publisher = {GitHub},
+  url = {https://github.com/rpy2/rpy2}
+}
+
+@software{pandasdevelopmentteam_2020_pandasdev,
+  title = {{{pandas-dev/pandas}}: {{pandas}}},
+  author = {Pandas Development Team},
+  year = {2020},
+  month = feb,
```

Comment:
```suggestion
@software{pandasdevelopmentteam_2024_pandasdev,
  title = {{{pandas-dev/pandas}}: {{pandas}}},
  author = {{The Pandas Development Team}},
  year = {2024},
  month = apr,
```

Updating date and fixing the rendering of author. DOI is already to "all versions".